modules/registry-operator-configuration-resource-overview.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * assembly/registry
+// * openshift_images/configuring-registry-operator.adoc
+
+
+[id="registry-operator-configuration-resource-overview-{context}"]
+= Image Registry Operator configuration
```

Comment:
If this module's just to hold this table, since the CA stuff should be a procedure, maybe
> Image registry Operator configuration parameters